sycl/test/regression/compile_on_win_with_mdd.cpp

```diff
@@ -0,0 +1,20 @@
+// REQUIRES: windows
+
+// RUN: %clang_cl -fsycl /EHsc /MDd -QMT %t.out /Fo%t.obj -c %s
+// RUN: %clang_cl -fsycl %t.obj -o %t.out
+// RUN: %t.out
+
+// The test aims to prevent regressions similar to one which caused by
+// https://github.com/intel/llvm/pull/12793.
+// If regression appears, the test should fail with abort() message during
+// application run.
+
+#include <sycl/sycl.hpp>
+
+#include <iostream>
```

Comment:
Isn't it extra? Maybe it's better to use <stdio.h>? Or check return code?

Reply:
I think it's OK: if user runs the test manually, they will see that test passes if there is `passed` in stdout